modules/go-health-monitoring.adoc

```diff
@@ -17,7 +17,9 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 
 . Click on the application name from the list to view the details of a specific application.
 
-. If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
+. If the application is out of sync or degraded, icons are displayed under *Resources* section. Hover over the icons to see health status and sync status. The icons are:
 
-.. For `degraded`, the broken heart icon is displayed.
-.. For `out of sync`, the yellow yield icon is displayed.
\ No newline at end of file
+.. For degraded status, the broken heart icon is displayed.
+.. For out of sync status, the yellow yield icon is displayed.
```

Comment:
Let's convert this descriptive information into a conditional step and be more specific about what the icons signify. How about something like this:
```suggestion
. If the *Resources* section displays icons, hover over the icons to get status details. 

** A broken heart indicates that resource issues have degraded the application's performance.
** A yellow yield sign indicates that resource issues have delayed data about the application's health.
```
Notes:
- Use articles to improve clarity and localization. ("IBM Style Guide").
- Use numbered steps in procedures to describe sequential actions.
- Use bullets to indicate non-sequential information.